backend/src/main/java/io/diveni/backend/controller/WebsocketController.java

```diff
@@ -37,11 +37,18 @@ public class WebsocketController {
 	@MessageMapping("/registerAdminUser")
 	public void registerAdminUser(AdminPrincipal principal) {
     LOGGER.debug("--> registerAdminUser()");
-		val session = ControllerUtils.getSessionOrThrowResponse(databaseService, principal.getSessionID());
+		var session = ControllerUtils.getSessionOrThrowResponse(databaseService, principal.getSessionID());
 		webSocketService.setAdminUser(principal);
 		if (session.getTimerTimestamp() != null) {
```

Comment:
what if it is `null`? No `else`?

Reply:
Then it directly throws an 404 exception
[Code](https://github.com/Sybit-Education/Diveni/blob/main/backend/src/main/java/io/diveni/backend/controller/ControllerUtils.java#L20)